packages/config/src/headers.js

```diff
@@ -11,25 +11,25 @@ const { warnHeadersParsing } = require('./log/messages')
 // Add `config.headers`
 const addHeaders = async function (config, logs) {
   const headersPath = resolve(config.build.publish, HEADERS_FILENAME)
+  const configWithHeaders = await addConfigHeaders(config, headersPath, logs)
```

Comment:
This file just moves the `try`/`catch` block handling headers parsing errors so it covers both `addHeaders()` and `addConfigHeaders()` since both are exported. Otherwise, exceptions in `addConfigHeaders()` are reported as system errors.